### PR TITLE
opensc-tool: Celarify usage on --send-apdu

### PR DESCRIPTION
**Pull request by frankmorgner**
##### Checklist

- [x] Documentation is added or updated
- [ ] New files have a LGPL 2.1 license statement
- [ ] PKCS#11 module is tested
- [ ] Windows minidriver is tested
- [ ] macOS tokend is tested

### CODE REVIEW

**Review by srsross — COMMENTED** [review]
The changes to `opensc-tool.1.xml` and `opensc-tool.c` for clarifying the usage of `--send-apdu` look good to me.  

Thanks, @frankmorgner.

File: src/tools/opensc-tool.c [hunk 1]
```diff
@@ -82,18 +82,18 @@ static const char *option_help[] = {
 	"Prints the ATR bytes of the card",
 	"Prints the card serial number",
 	"Identify the card and print its name",
-	"Get configuration key, format: section:name:key",
-	"Set configuration key, format: section:name:key:value",
+	"Get configuration, e.g. section:name:key",
+	"Set configuration, e.g. section:name:key:val",
```
Comment (Jakuje) [inline src/tools/opensc-tool.c:86]: I would keep the `value` for clarity.

File: src/tools/opensc-tool.c [hunk 2]
```diff
@@ -82,18 +82,18 @@ static const char *option_help[] = {
...
 	"Lists readers",
 	"Lists all installed card drivers",
 	"Recursively lists files stored on card",
-	"Sends an APDU in format AA:BB:CC:DD:EE:FF...",
+	"Sends an APDU (may need '-c default')",
 	"Uses reader number <arg> [0]",
 	"Does card reset of type <cold|warm> [cold]",
-	"Forces the use of driver <arg> [auto-detect; '?' for list]",
+	"Forces a card driver (use '?' for list)",
```
Comment (Jakuje) [inline src/tools/opensc-tool.c:93]: I would clarify this a bit more with something like " By default, the driver is detected as usual with opensc".

**Review by Jakuje — APPROVED** [review]

**Comment by frankmorgner** [comment]
@Jakuje Since we're breaking the limit of 80 characters elsewhere anyway, I don't insist on the shortened text. I can remove it from the changeset if you wish.

Some day, I may find the time to convert all the help texts to gengetopt, which handles this automatically...

**Comment by Jakuje** [comment]
> @Jakuje Since we're breaking the limit of 80 characters elsewhere anyway, I don't insist on the shortened text. I can remove it from the changeset if you wish.

Reading through that again, I think it is fine. The help text should be short and helpful. The complex explanations are better in the manual pages so disregard my last comment.

**Pull request merged**